contents/teams/sales-cs/objectives.mdx

```diff
@@ -1,13 +1,9 @@
-- **Objective**: git good at new sales
+- **Objective**: Scale Sales the PostHog way
 - **Goals**
-  - Increase revenue from new sales and/or expansion of existing customers:
-    - Add $1m in revenue from Extra Large customers - _Simon_
-    - Add $500k in revenue from Medium and Large customers - _Cameron_
-  - Get the sales engine working smoothly:
-    - Build v1 of our revops platform and kick off Salesforce implementation - _Mine_
-    - Hire 2x Account Executives and (maybe) an inbound SDR - _Charles_
-    - Create v1 of our sales playbook - _Charles_
-  - Have one big (input) number that you own and report publicly each week - _Everyone_
-- **Anti-goals**
-  - Spending lots of time working to retain customers who are steady state or unlikely to adopt >1 product.
-  - Events are not a goal. Instead, we should prioritize spending time in person with existing prospects.
+  - Increase the amount of revenue from customers paying $20k+ by 30% (Cameron, Leon, Seb, Simon)
+  - Get good at measuring expansion and predicting churn in managed vs. non-managed accounts (Simon)
+  - We understand the best ways to expand managed accounts and create a playbook (Cameron)
+  - 10x the sales cycle from initial call to close - look at each step and see how we can improve (Leon, Seb)
+  - Create a fully automated self serve process in Salesforce (Mine)
```

Comment:
```suggestion
  - Create a fully automated process for self serve customers in Salesforce (Mine)
```

Reply:
(Just to make it clear this is about customers, not us being able to self serve using SF)